### PR TITLE
Fix test DeleteVR -> PutVR

### LINKED ISSUE #178: Wrong test in visual_rules_test
The test for PUTing a VR is using DeleteVR in some of the cases
https://github.com/telefonicaid/perseo-fe/blob/master/test/component/visualrules_test.js#L335
https://github.com/telefonicaid/perseo-fe/blob/master/test/component/visualrules_test.js#L347

### PR DESCRIPTION
**Pull request by crbrox**
fixes #178 
@fgalan

### CODE REVIEW

File: test/component/visualrules_test.js [hunk 1]
```diff
@@ -330,9 +330,10 @@ describe('VisualRules', function() {
             ], done);
         });
         it('should return an error when core-endpoint is not working', function(done) {
+            var rule = utilsT.loadExample('./test/data/good_vrs/visual_rule_1.json');
             utilsT.setServerCode(400);
             utilsT.setServerMessage('what a pity!');
-            clients.DeleteVR('a very strange rule to exist', function(error, data) {
```
Comment (fgalan) [inline test/component/visualrules_test.js]: Reponedor/Cajera/Pescadería

File: test/component/visualrules_test.js [hunk 2]
```diff
@@ -330,9 +330,10 @@ describe('VisualRules', function() {
...
+            clients.PutVR(rule.name, rule, function(error, data) {
```
Comment (fgalan) [inline test/component/visualrules_test.js]: I understand that given that the fix is in a unit test it doesn't need CNR entry. Is that correct?

Just checking... :)

Comment (crbrox) [inline test/component/visualrules_test.js]: Correct, it's only a fix in tests. I don't think it deserves a CNR entry (there was not issue and it doesn't affect funcionality)

Comment (fgalan) [inline test/component/visualrules_test.js]: NTC

**Comment by fgalan** [comment]
LGTM

**Pull request merged**